src/main/java/net/openhft/chronicle/wire/BinaryWireHighCode.java

```diff
@@ -18,7 +18,23 @@
 package net.openhft.chronicle.wire;
 
 /**
- * These are the ranges of values to help decode the protocol.
+ * Enum to represent various high-level codes used in the binary wire protocol.
+ * Each constant in this enum serves as a marker to categorize the range of values
+ * used for decoding in the binary wire format. Although the enum itself does not
+ * contain any instances (indicated by the empty enum body), it does define several
+ * static final fields that serve as unique identifiers for each type of high-level code.
+ *
+ * <p><strong>High-Level Code Definitions:</strong></p>
+ * <ul>
+ *     <li>{@code END_OF_STREAM} (-1): Indicates the end of the data stream.</li>
```

Comment:
May be worth migrating these definitions next to each enum entry, e.g. https://stackoverflow.com/questions/6549821/how-to-javadoc-a-classs-individual-enums